docs/adding_new_keys.md

```diff
@@ -0,0 +1,131 @@
+# Adding a New Key
+
+Emission uses [`cocoapods-keys`](https://github.com/orta/cocoapods-keys) to store secrets (similar to a `.env` file). In order to expose these keys to our react-native components we must do a fair bit of setup.
+
+Links to examples below come from [this commit](https://github.com/artsy/emission/pull/1086/commits/4a2a3e9260e97d791536cf38376a06b0ad0946a8) adds a key for the Stripe API.
+
+## Steps
+
+#### 1. Add the key to the /Example app's Podfile.
+
+This is the extent of `cocoapods-keys` official setup, and after this you **could** set the key via `pod keys set <NAME>` or `pod install`... but we have more to do.
+[Example/Podfile](https://github.com/artsy/emission/blob/4a2a3e9260e97d791536cf38376a06b0ad0946a8/Example/Podfile#L63)
+
+```ruby
+plugin 'cocoapods-keys', {
+  :target => 'Emission',
+  :keys => [
+    'ArtsyAPIClientSecret',   # Authing to the Artsy API
+    'ArtsyAPIClientKey',      #
+    'StripePublishableKey',   # A new key!
+  ]
+}
+```
+
+---
+
+#### 2. Expose the keys in the `AREmission` module
+
+We'll need to update the `initWithUserId...` function, expose the new key as a property and add it to `constantsToExport`.
+
+[AREmission.h](https://github.com/artsy/emission/blob/4a2a3e9260e97d791536cf38376a06b0ad0946a8/Pod/Classes/Core/AREmission.h#L17-L34):
+
+```objc
+// ENV Variables
+@property (nonatomic, copy, readonly, nullable) NSString *stripePublishableKey; # Add this line
+
+# ...
+
+- (instancetype)initWithUserID:(NSString *)userID
+           authenticationToken:(NSString *)token
+                     sentryDSN:(nullable NSString *)sentryDSN
+          stripePublishableKey:(NSString *)stripePublishableKey # Add this line
```

Comment:
Same with this – totally fine to keep it as-is with `objc`, just pointing it out as an option.